client/a8c-for-agencies/sections/signup/signup-v2/components/multi-step-form/personalization/style.scss

```diff
@@ -0,0 +1,91 @@
+.signup-personalization-form {
+    .signup-multi-step-form__fields {
+        display: flex;
+        flex-direction: column;
+        overflow-y: auto;
+        flex: 1;
+    }
+
+    .signup-personalization-form__products-checkbox {
+        .multi-checkbox label {
+            flex: 0 1 calc(32% - 16px); // 3 columns for products
+
+            @media (max-width: 782px) {
+                flex: 0 1 calc(50% - 12px); // 2 columns on tablets
+            }
+
+            @media (max-width: 600px) {
+                flex: 0 1 100%; // Full width on mobile
+            }
+        }
+    }
+
+    .form-fieldset {
+        margin: 0;
+        padding: 2px;
+    }
+
+    // Form field styles
+    .a4a-form__section-field {
+        margin-bottom: 0;
+
+        &-label {
+            font-size: rem(14px);
+            font-weight: 600;
+            color: var(--studio-gray-80);
```

Comment:
It is better to avoid variables from the studio everywhere on this file, and instead, we should use --color-neutral. This will ensure we are using the color based on the theme.